go.mod

```diff
@@ -1,7 +1,12 @@
 module github.com/nats-io/nats.go
 
 require (
+	github.com/golang/protobuf v1.4.0
 	github.com/nats-io/jwt v0.3.2
+	github.com/nats-io/nats-server/v2 v2.1.6
 	github.com/nats-io/nkeys v0.1.4
 	github.com/nats-io/nuid v1.0.1
+	google.golang.org/protobuf v1.21.0
 )
+
+go 1.13
```

Comment:
Not completely sure of the impact of adding this one yet too...

Reply:
I will reset the go.mod/sum changes, but will be easier for me to do a git push force, so be aware.